### PR TITLE
Close #30

### LINKED ISSUE #30: Make game more responsive
Game would look better, if game board would be bigger on large screens.

### PR DESCRIPTION
**Pull request by albertodeago**
Improve responsiveness of the board.
Now it scales with user's viewport.

### CODE REVIEW

**Comment by indremak** [comment]
@albertodeago Thank you for PR 🙂 I will review it today.

**Comment by indremak** [comment]
@albertodeago great work 🙂 there is one bug though, when cells get bigger snake's eyes is not in the center. Could you fix it?

**Comment by albertodeago** [comment]
should be better now. I also added a media query to better center them in mobile and in desktop views

File: style.css [hunk 1]
```diff
@@ -78,11 +78,22 @@ button {
...
+    top: 11px;
+    left: 8px;
+  }
+  .snake-head__first-position::after {
+    top: -5px;
+    left: 8px;
+  }
+}
+
 .snake-head__second-position::before {
```
Comment (indremak) [inline style.css:97]: You should also change `second-position` before and after elements.

**Comment by indremak** [comment]
@albertodeago now eyes look better when snake goes to left or right 👍  but you should also change second position, that it would look better when snake is going top or bottom.

**Comment by albertodeago** [comment]
I noticed like 2 minutes before you wrote this 😄

**Pull request merged**